### PR TITLE
CAMEL-9808: SFTP: Enable configuration of bulk requests

### PR DESCRIPTION
**Pull request by thokuest**
This pull request makes it possible to configure the number of bulk requests issued by JSch. Increasing the number of bulk requests may slightly improve transfer speed at the cost of memory usage.

Usage:

    sftp://nosuchhost/workingdir?bulkRequests=64&...

See [CAMEL-9808](https://issues.apache.org/jira/browse/CAMEL-9808) for details.

### CODE REVIEW

File: components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/SftpConfiguration.java [hunk 1]
```diff
@@ -65,6 +65,8 @@
     private String preferredAuthentications;
     @UriParam(defaultValue = "WARN")
     private LoggingLevel jschLoggingLevel = LoggingLevel.WARN;
+    @UriParam(label="advanced", description="Specifies how many requests may be outstanding at any one time.")
```
Comment (davsclaus) [inline components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/SftpConfiguration.java]: Only document one place, eg the javadoc. So remove the description. Also put space around label="ad..."

File: components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/SftpConfiguration.java [hunk 2]
```diff
@@ -269,4 +271,16 @@ public LoggingLevel getJschLoggingLevel() {
...
+     * Specifies how many requests may be outstanding at any one time. Increasing this value may
+     * slightly improve file transfer speed but will increase memory usage.
+     */
+    public void setBulkRequests(Integer bulkRequests) {
+		this.bulkRequests = bulkRequests;
+	}
+    
+    public Integer getBulkRequests() {
+		return bulkRequests;
+	}
```
Comment (davsclaus) [inline components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/SftpConfiguration.java]: Make sure the code passes checkstyle, run

mvn compile -Psourcecheck

to validate

**Comment by thokuest** [comment]
Thanks for the peer review. Checkstyle violations have been resolved.

**Comment by davsclaus** [comment]
Thanks for the PR it has been merged. Do you mind closing this?

**Comment by thokuest** [comment]
Thanks for accepting the change.

**Pull request closed without merging**